### PR TITLE
Return a slightly more verbose error when "go get" fails.

### PR DESCRIPTION
**Pull request by mml**

### CODE REVIEW

File: hack/e2e.go [hunk 1]
```diff
@@ -145,7 +145,7 @@ func (t tester) getKubetest(get bool, old time.Duration) (string, error) {
 	}
 	log.Print("Updating kubetest binary...")
 	if err = t.wait("go", "get", "-u", "k8s.io/test-infra/kubetest"); err != nil {
-		return "", err // Could not upgrade
+		return "", fmt.Errorf("go get -u k8s.io/test-infra/kubetest: %v"), err // Could not upgrade
```
Comment (rmmh) [inline hack/e2e.go]: could put command in a []string to avoid a future change making the error message misleading, but it's probably fine

Comment (cblecker) [inline hack/e2e.go]: I think `err` needs to move inside of the `fmt.Errorf()`

**Comment by cblecker** [comment]
@mml:
```
W0629 17:18:30.637] # k8s.io/kubernetes/hack
W0629 17:18:30.638] hack/e2e.go:148: type string is not an expression
W0629 17:18:30.638] hack/e2e.go:148: not enough arguments in call to t.wait
W0629 17:18:30.638] 	have (<T>...)
W0629 17:18:30.638] 	want (string, ...string)
```

**Comment by mml** [comment]
@cblecker thanks... this one actually compiles and runs.

**Comment by mml** [comment]
@k8s-bot oh i'm missing a stupid @ sign... you could at least say so!

**Comment by ixdy** [comment]
/lgtm

actually fixing this so it DTRT if you had `test-infra` checked out would be even better, though I'm not sure how you'd do that. `go install` instead of `go get`, maybe?

**Comment by ixdy** [comment]
(OTOH, that would cause you to install an old version of `kubetest` if your `test-infra` workspace was out of date. hm.)

**Comment by mml** [comment]
OMG that test.

**Comment by rmmh** [comment]
golint wants s/eu_verbose/euVerbose/.

**Comment by mml** [comment]
> golint wants s/eu_verbose/euVerbose/.

@rmmh fixed PTAL

**Pull request merged**